tasks/document-ocr-fn/resources/function/function.py

```diff
@@ -22,6 +22,8 @@ def response(requestData: dict[str, Any]) -> dict[str, Any]:
 
     predictedMask = detect_document.run(segmentationModel, imagePath)
     predictedMask = processMask(predictedMask)
+    if predictedMask is None:
+        return functions.badRequest("Failed to determine document borders")
```

Comment:
Let's change message to something easily understandable like: "Failed to find document on the provided image". This is not necessarily a bad request if the provided image has a document but model failed to detect it. Is there a better HTTP status code to use for when our model did not find a document on a valid image?

Reply:
How about we use HTTP status code 404 "Not found"

Reply:
Maybe 500 would be enough - `Internal server error`.